src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  background-color: $color-white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  width: 200px;
+  height: 408px;
+  margin-right: 48px;
+  margin-top: 50px;
+  margin-bottom: 46px;
+
+  &__img {
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__text {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+    color: $color-dark;
+  }
+
+  &__code {
+    white-space: nowrap;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-grey;
+    margin-bottom: 16px;
+  }
+
+  &__stars {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__review {
+    font-weight: 400;
+    font-size: 10px;
+    margin-left: 13px;
+    color: $color-dark;
+  }
+
+  &__title {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    font-weight: 400;
+    font-size: 12px;
+    color: $color-grey;
+    margin-right: 85px;
+    line-height: 18px;
+  }
+
+  &__cost {
+    font-weight: 700;
+    font-size: 16px;
+    color: $color-dark;
+    line-height: 18px;
+  }
+
+  &__button {
+    border-radius: 5px;
+    background-color: $color-blue;
+    padding: 12px 70px 12px 69px;
+    text-transform: uppercase;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-align: center;
+    text-decoration: none;
+    color: $color-white;
+
```

Comment:
Add a `transition` for hover effect. It will be more user friendly 
([link](https://developer.mozilla.org/en-US/docs/Web/CSS/transition))

Sth like this `transition: color 250ms linear`